handlers/middlewares.go

```diff
@@ -56,7 +65,26 @@ func (h *Handler) AuthMiddleware(next http.Handler) http.Handler {
 			// }
 			// return
 			if provider.GetProviderType() == models.RemoteProviderType {
-				provider.Logout(w, req)
+				if err.Error() == "http: named cookie not present" {
+					provider.Logout(w, req)
+				}
+
+				graphQlError := []Body{{"expired"}}
```

Comment:
What is this doing?

Reply:
No, we are not touching graphQL, (just for sake of the name `graphQlError`) I will change the name.
I am sending `{
          "errors": {[
                "error" : "expired"
            ]}

}
` as json to get it when we do a query when token becomes invalid.

Reply:
I think giving more verbose error will help, like `auth token expired`

Reply:
done